### PR TITLE
Corrected ordering that tasks appear in when user enters multiple tasks

### PR DESCRIPTION
**Pull request by TheHollidayInn**
I reversed the array that holds multiples tasks before sending the tasks to the server so that the tasks will appear in the order the users enters them. 

https://github.com/HabitRPG/habitrpg/issues/4935

screen shot 2015-04-07 at 2 50 27 pm

screen shot 2015-04-07 at 2 50 31 pm

### CODE REVIEW

File: website/public/js/controllers/tasksCtrl.js [hunk 1]
```diff
@@ -38,7 +38,10 @@ habitrpg.controller("TasksCtrl", ['$scope', '$rootScope', '$location', 'User','N
     $scope.addTask = function(addTo, listDef) {
       if (listDef.bulk) {
         var tasks = listDef.newTask.split(/[\n\r]+/);
+        //Reverse the order of tasks so the tasks will appear in the order the user entered them
+        tasks.reverse();
         _.each(tasks, function(t) {
+          console.log(t)
```
Comment (crookedneighbor) [inline website/public/js/controllers/tasksCtrl.js]: Please remove this console.log

**Comment by TheHollidayInn** [comment]
Eh, how embarrassing. Thanks for catching that.

**Comment by crookedneighbor** [comment]
No worries. I've committed worse :)

**Comment by crookedneighbor** [comment]
@TheHolidayInn and @chimericdream, now that we have a passing build (FINALLY), I'd love for some tests to be written for this function. Are one of you up for that or should I put it on my list of things to do?

**Comment by chimericdream** [comment]
I may not get to it this evening, but I should be able to give it a go tomorrow if someone else doesn't finish it sooner.

**Comment by TheHollidayInn** [comment]
Woot! I'm down for testing. However, I'm having issues getting the testing working on my local install. Should I message somewhere else about that?

**Comment by crookedneighbor** [comment]
@TheHollidayInn What's going on when you try to run `npm test`?

**Comment by crookedneighbor** [comment]
@chimericdream I've merged this in, so if you write tests, you can write them against develop instead of this PR. Figured the core functionality of the function hasn't changed from what's in production, so why wait?

**Comment by TheHollidayInn** [comment]
Sounds good!

Here is the error I am receiving:

```
ERROR - Unable to start a WebDriver session.

/Users/krh121791/habitrpg/node_modules/protractor/node_modules/selenium-webdriver/lib/atoms/error.js:113
  var template = new Error(this.message);
                 ^
UnknownError: Unable to connect to host 127.0.0.1 on port 7055 after 45000 ms. Firefox console output:
```

**Comment by gisikw** [comment]
Make sure you have a java runtime installed!

Also, I should make that bit more forgiving. It's set up to run the full test suite if some portions fail...but selenium is part of the setup. Derp!

**Comment by TheHollidayInn** [comment]
Well, that fixed it... Thanks!

**Pull request merged**